core/test-api/src/main/kotlin/testApi/testRunner/TestRunner.kt

```diff
@@ -191,12 +191,11 @@ public abstract class AbstractTest<M : TestMethods, T : TestBuilder<M>, D : Dokk
             ?.replaceAfter(".jar", "")
     }
 
-    protected val commonStdlibPath: String? by lazy {
-        // TODO: feels hacky, find a better way to do it
-        ClassLoader.getSystemResource("kotlin/UInt.kotlin_metadata")
-            ?.file
-            ?.replace("file:", "")
-            ?.replaceAfter(".jar", "")
+    protected val commonStdlibPath: String?  by lazy {
+        // `kotlin-stdlib-common` is legacy
+        // we can use any platform dependency
+        // since common code should be resolved with all platform
+        jvmStdlibPath
```

Comment:
Can we remove/deprecate the existing paths (`jvmStdlibPath`, `jsStdlibPath`, `commonStdlibPath`), and just have a single `stdlibPath` instead?

Reply:
No, `jvmStdlibPath` and `jsStdlibPath` contain platform-specific declarations.

Reply:
Oh, I see what you mean. I misunderstood the comment as "you can use any platform stdlib in any scenario (even use jvm stdlib in js)", but you meant it as "any platform stdlib can be used as a 'common' stdlib"